test/unit/core/product-transformer.ts

```diff
@@ -245,52 +246,18 @@ suite('Product Transformation', ({ expect, spy, stub }) => {
     });
 
     describe('transformPrice()', () => {
-      it('should return empty string on undefined input', () => {
-        const result = TransformUtils.transformPrice(undefined);
-
-        expect(result).to.eq('');
-      });
-
-      it('should format string', () => {
-        const result = TransformUtils.transformPrice('12.0');
-
-        expect(result).to.eq('12.00');
-      });
-
-      it('should format string of european formatting', () => {
-        const result = TransformUtils.transformPrice('12,0');
-
-        expect(result).to.eq('12,00');
-      });
-
-      it('should not format if regex does not match', () => {
-        const price = '12';
-
-        const result = TransformUtils.transformPrice(price);
-
-        expect(result).to.eq(price);
-      });
-    });
-
-    describe('simplePad()', () => {
-      it('should pad string of length 0', () => {
-        const result = TransformUtils.simplePad('');
-
-        expect(result).to.eq('00');
-      });
-
-      it('should pad string of length 1', () => {
-        const result = TransformUtils.simplePad('5');
-
-        expect(result).to.eq('50');
-      });
-
-      it('should not pad string of length greater than 1', () => {
-        const input = '1234';
-
-        const result = TransformUtils.simplePad(input);
-
-        expect(result).to.eq(input);
+      it('should use the currency-formatter library', () => {
```

Comment:
I don't think you should assert this because it's an implementation detail. You should instead assert instead that the function that uses it outputs the correct format and not stub `CurrencyFormatter`.

Reply:
You are correct, thanks